base.mk

```diff
@@ -50,7 +51,17 @@ PRODUCT_COPY_FILES += \
 
 # LineageOS build may need this to make NFC work
 PRODUCT_PACKAGES += \
-        NfcNci  
+        NfcNci \
+	libnfc-nci \
+    	Tag \
+    	com.android.nfc_extras \
```

Comment:
Well, as far as I know, those are unrelated to actual NFC support.
Tag is to react automatically on specific NFC tags and nfc_extras is for some extra NFC features.

I'm totally ok with integrating them though. But that's not related to Huawei support